python/dlr/libpath.py

```diff
@@ -48,4 +51,6 @@ def find_lib_path():
         raise DLRLibraryNotFound(
             'Cannot find DLR Library in the candidate path, ' +
             'List of candidates:\n' + ('\n'.join(dll_path)))
-    return lib_path
+
+    # If multiple paths are found always prefer the one that is in system paths.
+    return lib_path[0]
```

Comment:
We do not need to check all `dll_path` for existence (on line 50) `lib_path = [...]